--- /home/commlab/Documents/Gateway_POC-test/class_test.py	2021-09-29 11:02:49.306610194 +0530
+++ /home/commlab/Documents/Gateway_POC-test/OTA/python_app.py	2021-09-29 10:51:26.681408819 +0530
@@ -1,7 +1,5 @@
-#New Code
-
 num = 29
-#comment 1
+
 flag = False
 
 if num > 1:
@@ -12,12 +10,10 @@
             flag = True
 
             break
-print("New Line....")
+
 if flag:
     print(num, "is not a prime number")
 else:
     print(num, "is a prime number")
 
-#Extra Added
-print("Hello World!")
-print("one more line")
+
